### PR TITLE
Static translations 0

### PR DESCRIPTION
**Pull request by raymondlee00**

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -66,7 +66,6 @@ flutter:
   # To add assets to your application, add an assets section, like this:
   assets:
     - images/
-    - lang/
     - assets/
```
Comment (raymondlee00) [inline pubspec.yaml]: add in "-i18n/"

File: i18n/bn-BN.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    "pleaseEnterATitle": "একটি শিরোনাম লিখুন দয়া করে",
+    "pleaseEnterADescription": "একটি বিবরণ প্রবেশ করুন",
+    "newPost": "নতুন পোস্ট",
+    "createPost": "পোস্ট তৈরি করুন",
+    "commentSection": "মন্তব্য বিভাগ",
+    "enterAComment": "একটি মন্তব্য লিখুন",
+    "searchForum": "ফোরাম অনুসন্ধান করুন",
+    "signOut": "সাইন আউট",
+    "userDoesNotExist": "ইমেল User ইমেল with সহ ব্যবহারকারীর অস্তিত্ব নেই",
+    "incorrectPassword": "ভুল পাসওয়ার্ড৤",
```
[the remark this answers is not in the document]
Comment (ericlau00) [reply i18n/bn-BN.json]: i cannot see this character on my chrome browser on desktop but i was able to see it on mobile

File: lib/auth/sign_in.dart [hunk 3]
```diff
@@ -101,11 +100,12 @@ class _LoginPageState extends State<LoginPage> with Screen {
 
   Future<dynamic> signIn() async {
     final formState = _formKey.currentState;
-    if (formState.validate()) {
+    if (formState.validate()) { // TODO: fix these nested if statements
```
[the remark this answers is not in the document]
Comment (ericlau00) [reply lib/auth/sign_in.dart]: i think this will have to suffice

File: lib/auth/sign_in.dart [hunk 4]
```diff
@@ -101,11 +100,12 @@ class _LoginPageState extends State<LoginPage> with Screen {
...
       formState.save();
       final responseMessage = await APIConnect.loginUser(_email, _password);
       if (responseMessage["message"] == "Error") {
-        showToast(responseMessage["error"]["errorMessage"], _scaffoldKey);
+        String message = responseMessage["error"]["errorMessage"] == "Incorrect password" ? I18n.of(context).incorrectPassword : I18n.of(context).userDoesNotExist(_email); // TODO: need a better to code for this logic
+        showToast(message, _scaffoldKey);
```
[the remark this answers is not in the document]
Comment (ericlau00) [reply lib/auth/sign_in.dart]: are these the only two errors that the api provide? @kazijamal

File: lib/auth/sign_in.dart [hunk 5]
```diff
@@ -101,11 +100,12 @@ class _LoginPageState extends State<LoginPage> with Screen {
 
   Future<dynamic> signIn() async {
     final formState = _formKey.currentState;
-    if (formState.validate()) {
+    if (formState.validate()) { // TODO: fix these nested if statements
       formState.save();
       final responseMessage = await APIConnect.loginUser(_email, _password);
       if (responseMessage["message"] == "Error") {
-        showToast(responseMessage["error"]["errorMessage"], _scaffoldKey);
+        String message = responseMessage["error"]["errorMessage"] == "Incorrect password" ? I18n.of(context).incorrectPassword : I18n.of(context).userDoesNotExist(_email); // TODO: need a better to code for this logic
```
Comment (ericlau00) [inline lib/auth/sign_in.dart]: split ternary line into several lines

Comment (kazijamal) [reply lib/auth/sign_in.dart]: yes those are the only two api errors on login

File: i18n/bn-BN.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (raymondlee00) [reply i18n/bn-BN.json]: cannot load character on kaz's android phone

File: i18n/bn-BN.json [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+    "account": "হিসাব",
+    "proposals": "প্রস্তাব",
+    "endsInNumDays": "{numDays} দিনগুলিতে শেষ হয়",
+    "yes": "হ্যাঁ",
+    "no": "না",
+    "numYes": "{numYes} হ্যাঁ",
+    "numNo": "{numNo} no",
+    "numVotes": "{numVotes} ভোট",
+    "votingOnProposalEnds": "প্রস্তাবনায় ভোট শেষ {শেষ সময়}",
+    "youHaveVoted": "আপনি {VotVal vot ভোট দিয়েছেন!",
```
[the remark this answers is not in the document]
Comment (raymondlee00) [reply i18n/bn-BN.json]: doesnt work

File: i18n/bn-BN.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (raymondlee00) [reply i18n/bn-BN.json]: changed "ভুল পাসওয়ার্ড৤" to "ভুল গোপন নম্বর" which means "wrong password" in bangla in both bn-BN.json and lib/generated/i18n.dart

File: i18n/bn-BN.json [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+    "attending": "অংশগ্রহণ করছেন?",
+    "pleaseEnterATitle": "একটি শিরোনাম লিখুন দয়া করে",
+    "pleaseEnterADescription": "একটি বিবরণ প্রবেশ করুন",
+    "newPost": "নতুন পোস্ট",
+    "createPost": "পোস্ট তৈরি করুন",
+    "commentSection": "মন্তব্য বিভাগ",
+    "enterAComment": "একটি মন্তব্য লিখুন",
+    "searchForum": "ফোরাম অনুসন্ধান করুন",
+    "signOut": "সাইন আউট",
+    "userDoesNotExist": "ইমেল User ইমেল with সহ ব্যবহারকারীর অস্তিত্ব নেই",
```
[the remark this answers is not in the document]
Comment (raymondlee00) [reply i18n/bn-BN.json]: changed "ইমেল User ইমেল with সহ ব্যবহারকারীর অস্তিত্ব নেই" to "এই ইমেল সহ ব্যবহারকারী বিদ্যমান নেই" which means "user with this email does not exist" in english

File: i18n/ja-JP.json [hunk 8]
```diff
@@ -0,0 +1,70 @@
+{
+    "greetTo": "こんにちは{name}",
+    "signIn": "サインイン",
+    "signUp": "サインアップ",
+    "email": "Eメール",
```
[the remark this answers is not in the document]
Comment (raymondlee00) [reply i18n/ja-JP.json]: nope, just japanese lolol

File: i18n/bn-BN.json [hunk 9]
```diff
@@ -0,0 +1,70 @@
+{
+    "greetTo": "হ্যালো {নাম}",
```
[the remark this answers is not in the document]
Comment (raymondlee00) [reply i18n/bn-BN.json]: this field aint needed anyways

**Pull request closed without merging**